website/docs/feature-flag-tutorials/use-cases/security-compliance.md

```diff
@@ -0,0 +1,239 @@
+
+From an architectural perspective, we implement this by allowing Unleash users to define cross-origin resource sharing (CORS) policies to prevent unwarranted access to your feature flag system. From a compliance perspective, this demonstrates that your organization leverages a feature flag system that takes a multi-faceted approach to security.
+
+![An image of a CORS section in Unleash Admin UI for adding domains that can connect to Unleash Frontend API.](/img/use-case-security-cors.png)
+
+In the Unleash Admin UI, we have a cross-origin resource sharing (CORS) section with a form so you can list domains that will be allowed to call our Frontend API. This is an access control mechanism that gives you control over which domains interact with Unleash. Different CORS policies can be applied to the development, staging/test, and production environments you configure.
+
+By default, this CORS origin list includes an asterisk (\*), which means that all services that call the Frontend API will be allowed access. You can edit this form to include only your specific domains.
+
+### Set up IP allow lists for enhanced security
+
+Enhance your network security with IP allow lists, which give your organization access control based on specific network parameters. This feature restricts all access to Unleash to only be used by certain IP addresses. For example, if you’re using company VPNs or servers, you can add those IP addresses to the list to ensure they can access Unleash.
+
+Next, we’ll explore change management as an important tool to further enhance access control mechanisms and make your workflows auditable.
+
+## Use a change management workflow for auditing
+
+Engineers typically go through a development workflow where they write code, submit their code changes for review, get explicit approval from other engineers, and merge their approved changes into the main branch. This occurs in a version control system like Git/Github. In the context of feature flag management, this approach is similar to how you can submit [change requests](/reference/change-requests) in Unleash to make updates to resources in your projects across your environments.
+
+Think of change requests as _more_ than just a tool to sign off on what’s happening in your feature flag system. It’s a way to ensure multiple layers of security for your teams.
+
+For large organizations in highly regulated industries, it’s important to keep track of changes in your feature flag system. When your team members are using Unleash, consider that while some users have permission to modify flags, you can still put guardrails in place to approve or deny the changes.
```

Comment:
I think the part of auditing is covered in another section, this change would make it closer to change request and 4 eyes approval. 
```suggestion
For large organizations in highly regulated industries, it’s important to have a second pair of eyes validating a change. When your team members are using Unleash, consider that while some users have permission to modify flags, you can still put guardrails in place to approve or deny the changes.
```